### ISSUE TITLE
[BUG] Navigation buttons disappear

### ISSUE THREAD

**Issue by kraves-oz** [bitfocus/companion#2927]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Standard page with navigation, except have replaced title with regular button.
After some time, perhaps within couple hours sitting idle, the arrows disappear and buttons are blank.
changing pages and back again the arrows return normally.

Anyone seen thus behaviour before?  It's a new one for me.

Kyle.

### Environment (please complete the following information)

```markdown
- OS: Win 11 23H2
- Browser: Chrome 126.0.6478
- Companion Version: 3.3.1-7001-Stable
- Standard 15 key SD, relatively early release
```

**Comment by mvaneijken** [comment]
Same here, buttons turn black:
![image](https://github.com/bitfocus/companion/assets/11209460/89d31c04-f762-404d-b1d8-7dc4860acc10)

![image](https://github.com/bitfocus/companion/assets/11209460/8a35becd-91da-4536-8c2b-5ee1de6e36a3)

Restarting the Companion does not help.

Same OS and version as above: 3.3.1 (3.3.1+7001-stable-ee7c3daa)

**Comment by Julusian** [comment]
@mvaneijken your issue sounds different. And yours sounds like some known issues with emoji (see #2832). Unfortunately this is not simple to fix with the current drawing library, so won't see a fix before 3.4. The best advice is to remove the offending emoji then restart companion

**Comment by Oldman-Winter** [comment]
I just updated to 3.3.1 and the fonts look different from 3.0.0
Some text on Auto size now wraps one letter to the next line unnecessarily.
Some emojis are just different looking and are not a big deal.

However, many emojis break the button graphics all together. 🔀💯 These are two examples
The button just goes black and won't display anything at all until you remove the offending emojis and then restart companion.
